README.md

```diff
@@ -69,19 +68,6 @@
 
 
 
-
-
-
-<p align="right">(<a href="#readme-top">back to top</a>)</p>
-
-<!-- LIVE DEMO -->
-
- ## 🚀 Live Demo <a name="live-demo"></a> 
-
-<!-- [Live Demo Link]() -->
-
-
```

Comment:
- [ ] `License` section: I think it would be better if you fix the broken `[MIT](./LICENSE)` link in your README file to make it more professional as it shows a **404** error page once clicked. And to fix that, you can only create a new file name `LICENSE` in your root directory and get its content from this **[template](https://github.com/microverseinc/readme-template/blob/master/MIT.md)** or rename this `[MIT](./LICENSE)` to `[MIT](./MIT.md)`, then the created file in your directory name it `MIT.md` and use the same content in this **[template](https://github.com/microverseinc/readme-template/blob/master/MIT.md)**